buildbot/google/docker/buildbot-clangd-ubuntu-clang/Dockerfile

```diff
@@ -53,8 +45,17 @@ RUN wget -O - https://apt.kitware.com/keys/kitware-archive-latest.asc 2>/dev/nul
     # Clean apt cache to reduce image size.
     apt-get clean
 
-# Install build bot (server was at 2.8.5-dev at time of writing).
-RUN pip3 install buildbot-worker==2.8.4
+# configure default versions of LLVM tools and python needed for
+# buildbot-worker. We deliberately do this after installing all the system
+# packages, as ubuntu 18.04 uses python3.6 by default.
```

Comment:
NIT: I was a little confused when I first read this (why can't Ubuntu use a newer version rather than the default?)
I suspect this could be avoided if you say `..., as apt-get install breaks if we change the Python from default to 3.8`